spec/services/meal_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe Meal do
+  let(:params) do
+    { params: { name: 'pizza',
+                course: 'main course',
+                price: 42 },
+      menu: Menu.create }
+  end
+
+  describe 'Creator' do
+    subject(:meal) { Meal::Creator.call(params) }
+    it 'creates a meal' do
+      is_expected.to be_a Meal
+    end
+
+    describe 'meal' do
+      it 'is a pizza' do
+        expect(meal.name).to eq('pizza')
```

Comment:
Try using declared variables instead of hardcoded strings so that if there's a need to change something you have to edit the code only in one place.
For example, what if I wanted to changed meal name from `pizza` to `burger`? I'd have to change it in 2 places instead of one. 

`expect(meal.name). to eq params[:name]`